src/drive/web/modules/views/OnlyOffice/Toolbar/FileName.jsx

```diff
@@ -27,8 +56,23 @@ const FileName = ({ fileWithPath }) => {
   )
 }
 
-FileName.propTypes = {
+DumbFileName.propTypes = {
   fileWithPath: PropTypes.object.isRequired
 }
 
+const mapDispatchToProps = dispatch => ({
+  rename: file => dispatch(startRenamingAsync(file))
+})
+
+const mapStateToProps = (state, ownProps) => ({
+  isRenaming:
+    isRenaming(state) &&
+    get(getRenamingFile(state), 'id') === ownProps.fileWithPath.id
+})
+
+const FileName = connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(DumbFileName)
```

Comment:
I'm a little bothered by this. We want to avoid the use of redux for this kind of thing. It is not easy to take this component out of drive because of the redux action. 

I understand you have done that way because the other component is written like that... So if you have time to rewrite this one without redux it can be cool, otherwise it doesn't matter, we'll do it later.

Reply:
Pas de souci je modifie 👍